plugins/parsers/csv/parser.go

```diff
@@ -169,6 +171,12 @@ func parseCSV(p *Parser, r io.Reader) ([]telegraf.Metric, error) {
 	for _, record := range table {
 		m, err := p.parseRecord(record)
 		if err != nil {
+			if p.SkipErrors {
+				if p.Log != nil {
+					p.Log.Infof("[parsers.csv] Malformed csv line: %s", err.Error())
+				}
+				continue
+			}
 			return metrics, err
```

Comment:
For consistency with above error message:
```suggestion
			return metrics, fmt.Errorf("parsing error: %w", err)
```